packages/renderer/src/validate-ffmpeg.ts

```diff
@@ -1,22 +1,39 @@
 import execa from 'execa';
 import {existsSync, statSync} from 'fs';
 import os from 'os';
-import {getFfmpegBuildInfo, getFfmpegVersion} from './ffmpeg-flags';
+import {
+	downloadBinary,
+	ffmpegInNodeModules,
+	getFfmpegBuildInfo,
+	getFfmpegDownloadUrl,
+	getFfmpegVersion,
+} from './ffmpeg-flags';
 import {warnAboutFfmpegVersion} from './warn-about-ffmpeg-version';
 
 const existsMap: {[key: string]: boolean} = {};
 
 export const binaryExists = (
-	name: 'ffmpeg' | 'brew',
-	localFFmpeg: string | null
+	name: 'ffmpeg' | 'ffprobe' | 'brew',
+	localExecutable: string | null
 ) => {
 	if (typeof existsMap[name] !== 'undefined') {
 		return existsMap[name];
 	}
 
-	if (name === 'ffmpeg' && localFFmpeg) {
+	if (name === 'ffmpeg' && localExecutable) {
 		try {
-			statSync(localFFmpeg);
+			statSync(localExecutable);
+			existsMap[name] = true;
+		} catch (err) {
+			existsMap[name] = false;
+		}
+
+		return existsMap[name];
+	}
+
+	if (name === 'ffprobe' && localExecutable) {
```

Comment:
The logic for ffprobe and ffmpeg appears to be the same -> just 1 if

Reply:
resolved in 417c039192d45204f123b8d9179c4d09c2c60311